dtfs-central-api/src/helpers/get-fee-record-payment-entity-group-reconciliation-data.ts

```diff
@@ -23,14 +23,17 @@ export const getFeeRecordPaymentEntityGroupReconciliationData = async (
 
   const reconciledFeeRecords = group.feeRecords.filter(({ status }) => status === FEE_RECORD_STATUS.RECONCILED);
 
-  if (reconciledFeeRecords.length === 0) {
+  if (reconciledFeeRecords.length !== group.feeRecords.length) {
```

Comment:
Can we add a comment/documentation here explaining why we need to do this?

Reply:
Just picked up this PR, I believe this is to ensure we only return the reconciliation data if all fee records in the group are reconciled. I've pulled this condition out into a self-describing constant to try and avoid the comment - is this ok? Thanks

Reply:
This also means we can simplify the filtering to be an `every` check (see diff), simplifying the function

Reply:
Nice!